new/LIT7041LordsPrayerAmh.xml

```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7041LordsPrayerAmh" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="am" xml:id="t1">የጌታ፡ ጸሎት፡</title>
+                <title xml:lang="am" type="normalized" corresp="#t1">Yagetā ṣalot</title>
+                <title xml:lang="en" corresp="#t1">Lord's prayer</title>
+                <title xml:lang="am" xml:id="t2">አባታችን፡ ሆይ፡ በሰማያት፡ የምትኖር፡</title>
+                <title xml:lang="am" type="normalized" corresp="#t2"></title>
+                <title xml:lang="en" corresp="#t2">Our Father, who dwells in heaven</title>
```

Comment:
please add ...

Reply:
I think, I should add both: (...) to the english title and the transcription:

ʾabbātāččǝn hoy ba-samāyāt ya-mǝtnor (?)

Reply:
If you listen, it oscilates between yammǝtǝnor and yammǝtnor (I would trasliterate it yammǝtǝnor). I never saw transliterated it as yammǝttǝnor (you can find examples of reading of this prayer online).